frontend/src/views/web/challenge/leaderboard.html

```diff
@@ -76,7 +76,7 @@ <h5 class="w-300">Leaderboard</h5>
                         </div>
                     </div>
                 </div>
-                <table ng-if="challenge.isResult && challenge.leaderboard.length" ng-attr-id="leaderboard_table" ng-class="['highlight', {'watermarked': challenge.showBackground}]">
+                <table ng-if="challenge.isResult && challenge.leaderboard.length" ng-attr-id="leaderboard_table" ng-class="['highlight', 'watermarked']">
```

Comment:
@matthew-so this will enable it for both prod and staging right? We only want this change on staging for now. Can you please fix the check?

Reply:
Right. I'll add showBackground back to challengeCtrl

Reply:
Fixed